trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'baseDecorator'
```

Comment:
- [ ] Great job so far @sumon766 There are just a few changes you need to make and you'll be on your way to the next milestone :wink: 
- [ ] I noticed here that the `require_relative` statement in the `trimmer_decorator.rb `file has the `base_decorator.rb` file spelt wrongly. As a result, when I tried to test if you managed to decorate a class successfully using the code provided in the project requirements, I got an error. I have pasted the error message below. So kindly fix this :+1: 
- Error message
![image](https://github.com/sumon766/ruby-oop/assets/102699958/2576e8c4-9cc5-4682-a6be-8cb97cf325f7)